lib/fastly/invoice.rb

```diff
@@ -4,19 +4,7 @@
 class Fastly
   # An invoice for a time period
   class Invoice < Base
-    attr_accessor :service_id, :service_name, :start_time, :end_time, :total, :regions
```

Comment:
@leklund it seems like this PR constitutes a breaking change so https://github.com/fastly/fastly-ruby/pull/127 should be a major bump.

Reply:
ah. yep. I'll re-tag